lib/storekit/receipt.rb

```diff
@@ -1,19 +1,42 @@
+require 'ostruct'
+
 module StoreKit
-  class Receipt
-    attr_reader :receipt_data
+  class Receipt < OpenStruct
+    TRANSFORM = {
+       'unique_identifier' => true,
+       'product_id' => true,
+       'bid' => true,
+       'unique_vendor_identifier' => true,
+       'bvrs' => true,
+       'transaction_id' => proc { |v| v.to_i },
+       'original_transaction_id' => proc { |v| v.to_i },
+       'quantity' => proc { |v| v.to_i },
+       'item_id' => proc { |v| v.to_i },
+       'web_order_line_item_id' => proc { |v| v.to_i },
```

Comment:
According documentation yes, as well as the time fields but that is in reality? They are ids as integers. Why don't you look into the gist?